srcpkgs/polari/template

```diff
@@ -1,12 +1,13 @@
 # Template file for 'polari'
 pkgname=polari
-version=45.0
+version=46.0
 revision=1
 build_style=meson
 build_helper="gir"
 hostmakedepends="pkg-config itstool gettext glib-devel gtk-update-icon-cache
  desktop-file-utils"
-makedepends="gjs-devel telepathy-logger-devel telepathy-glib-devel"
+makedepends="gjs-devel telepathy-logger-devel telepathy-glib-devel
+ tracker-devel"
 depends="telepathy-logger telepathy-glib libsecret gtk4 libadwaita
```

Comment:
remove `telepathy-logger`. read the changelog.

Reply:
and also add `tracker` to depends

Reply:
Ah, sorry
will update asap

Reply:
@chrysos349 Why tracker instead of libtracker? (the latter should get pulled in automatically)

Reply:
@oreo639 you're right about libtracker.